plenum/common/member/member.py

```diff
@@ -0,0 +1,24 @@
+from hashlib import sha256
+
+from plenum.common.constants import TXN_TYPE, NYM, TARGET_NYM, TXN_ID, ROLE, VERKEY
+from plenum.common.types import f
+
+
+class Member:
```

Comment:
I'm not sure I know enough about Plenum to have an opinion, I'll ping Jason but let not get hung up on something that could easily get refactored later.